lib/utils/client_manager.dart

```diff
@@ -45,23 +45,26 @@ abstract class ClientManager {
         ),
       );
     }
-    if (clients.length > 1 && clients.any((c) => !c.isLogged())) {
-      final loggedOutClients = clients.where((c) => !c.isLogged()).toList();
-      for (final client in loggedOutClients) {
-        Logs().w(
-          'Multi account is enabled but client ${client.userID} is not logged in. Removing...',
-        );
-        clientNames.remove(client.clientName);
-        clients.remove(client);
-      }
-      await Store().setItem(clientNamespace, jsonEncode(clientNames.toList()));
-    }
+    // if (clients.length > 1 && clients.any((c) => !c.isLogged())) {
+    //   final loggedOutClients = clients.where((c) => !c.isLogged()).toList();
+    //   for (final client in loggedOutClients) {
+    //     Logs().w(
+    //       'Multi account is enabled but client ${client.userID} is not logged in. Removing...',
+    //     );
+    //     clientNames.remove(client.clientName);
+    //     clients.remove(client);
+    //   }
+    //   await Store().setItem(clientNamespace, jsonEncode(clientNames.toList()));
+    // }
```

Comment:
Can you explain it more?

Reply:
When getClients from store, it will check the log out account, then remove it.

Reply:
Because it's the correct way to remove log out account, so i remove the comment and remove clientName in store when log out